modules/phase_field/src/materials/ComputePolycrystalElasticityTensor.C

```diff
@@ -48,9 +48,6 @@ ComputePolycrystalElasticityTensor::computeQpElasticityTensor()
   const std::vector<std::pair<unsigned int, unsigned int> > & active_ops = _grain_tracker.getElementalValues(_current_elem->id());
```

Comment:
Well, if that's not of much of a concern then I would not mind dropping  `getElementalValues`.